src/Store/CookieStore.php

```diff
@@ -201,11 +200,12 @@ public function destroy($session_id)
             $params['secure'],
             $params['httponly']
         );
+
+        return true;
     }
 
-    #[\ReturnTypeWillChange]
-    public function gc($maxlifetime)
+    public function gc(int $maxlifetime): int|false
```

Comment:
```suggestion
    public function gc(int $maxlifetime): bool
```
This never returns an int, and I'm pretty sure `false` as a return type is invalid for PHP 8.1 (https://www.php.net/releases/8.2/en.php#null_false_true_types)

Reply:
Also check others in this PR where `false` is used as a return type.

Reply:
I don't having a very specific return type here provides much value. The more generic return type gives us the flexibility to change our minds later on.

`int|false` is valid return type.